gempython/tools/amc_user_functions_xhal.py

```diff
@@ -401,6 +406,71 @@ def getTTCStatus(self, ohN, display=False):
                 print("\tENABLE: \t\t\t%i"%(running))
         return running
 
+    def getTriggerLinkStatus(self,printSummary=False, checkCSCTrigLink=False, ohMask=0xfff):
+        """
+        Gets the trigger link status
+        """
```

Comment:
The documentation here should describe how this will behave and how `ohMask` will be treated in each of the cases of `checkCSCTrigLink`.  See for example lines 474 to 484.

Reply:
ok I have updated the text here -- please have a look at this one again

Reply:
Lines 411 and 412 are fine.  But subsequent lines I think are still not quiet clear. I would say:

> ohMask is a 12 bit number, see checkCSCTrigLink for how each bits are interpreted.
> If checkCSCTrigLink is False the ohMask will define which OH's to query, a 1 in the N^th bit means to query the N^th optohybrid.
> If checkCSCTrigLink is True, the ohMask is expected to have all odd bits set to 0.  Only even bits may be nonzero.  If the N^th even bit is nonzero the trigger link status for that OHN and OHN+1 will be queried.  It is expected that the GEM trigger link from the physical optohybrid is going to the OHN fiber slot on this AMC and the CSC Trigger Link from the physical optohybrid is going to the OHN+1 fiber slot on this AMC. 

I think this would be more representative and clear.